build.sbt

```diff
@@ -55,6 +57,35 @@ ThisBuild / githubWorkflowPublish := Seq(
   )
 )
 
+// publish website
+
+ThisBuild / githubWorkflowAddedJobs += WorkflowJob(
+  id = "site",
+  name = "Deploy site",
+  needs = List("publish"),
+  javas = (ThisBuild / githubWorkflowJavaVersions).value.toList,
+  scalas = (ThisBuild / scalaVersion).value :: Nil,
+  cond = """
+  | always() &&
+  | needs.build.result == 'success' &&
```

Comment:
Couldn't you just remove these two lines? `always()` is to have this job run even when the dependencies failed, but you rule that out in the next line.

Reply:
Copy paste from the fs2 repo 😅 I'll remove it, you're right.